Project.toml

```
@@ -1,7 +1,7 @@
 name = "KernelAbstractions"
 uuid = "63c18a36-062a-441e-b654-da1e3ab1ce7c"
 authors = ["Valentin Churavy <v.churavy@gmail.com> and contributors"]
-version = "0.10.0-dev"
+version = "0.9.35"
```

```suggestion
version = "0.10.0-dev"
```
Revert to 0.10.0-dev before merging